### DISCUSSION TITLE
Add a "break row" UI element to field layout designer

### DISCUSSION THREAD

**Discussion by evanwarner** [craftcms/cms#8868] [Ideas]
The field layout designer's column setting is really nice for imposing a max-width on fields intended to have short values, but sequential narrow fields will automatically become side-by-side, and I don't always want that. Could the solution lie in an UI element in the layout designer that fills the remaining columns of a row?

**Comment by evanwarner** [comment]
I suppose alternatively, allowing fields to have a max-width settings (like how Number fields have a **Size** setting) could fix this as well. I think that's what #1279 is suggesting.

**Comment by furioursus** [comment]
Yeah, this is great. The horizontal rule ui element is a great stopgap solution but I’d love to see something more transparent.
